test/storage.test.js

```diff
@@ -20,8 +20,10 @@ const mockFs = require('mock-fs');
 
 const {getSource, putRendition, getAsset} = require('../lib/storage');
 const nock = require('nock');
+const mockRequire = require("mock-require");
 const fs = require('fs-extra');
 const path = require('path');
+require('dotenv').config({ path: path.join(__dirname, '.env') });
```

Comment:
Not needed if all tests are mocked (and they should be, either mocked or skipped at least if unmocked).